includes/class-sensei-analysis.php

```diff
@@ -905,15 +905,22 @@ public function report_set_headers( $filename = '' ) {
 	 * @return object                 class instance object
 	 */
 	public function load_report_object( $name = '', $data = 0, $optional_data = null ) {
-		$object_name = 'Sensei_Analysis_' . $name . '_List_Table';
-		if ( is_null( $optional_data ) ) {
-			$sensei_analysis_report_object = new $object_name( $data );
+		if ( 'Overview' === $name ) {
```

Comment:
Hmm, can't get it... I'm sure we did it earlier: https://github.com/Automattic/sensei/commit/cb31337a9f7e19b8d59290143727d8182a9d7580#diff-8be84703e6d7b31882b67392181926192bd7175d46cd63c5685bf461f72eaf9f

Looks like we reverted those changes at some point... :(